lib/rubocop/cop/style/redundant_line_continuation.rb

```diff
@@ -115,6 +115,14 @@ def redundant_line_continuation?(range)
           parse(source.gsub(/\\\n/, "\n")).valid_syntax?
         end
 
+        def inside_string_literal?(range, token)
+          ALLOWED_STRING_TOKENS.include?(token.type) && token.pos.overlaps?(range)
+        end
+
+        def continue_identifier?(current_token, next_token)
```

Comment:
I don't quite get what's the significance of having two consecutive identifiers. Isn't it more important what those identifiers signify (e.g. a method name and one of its arguments or whatever)?